ipv8/attestation/trustchain/block.py

```diff
@@ -1,12 +1,10 @@
-from hashlib import sha256
```

Comment:
I used Ctrl + Alt + O for import optimization in PyCharm. It sometimes scrambles the order. Going back to the issue with `....attestation.trustchain.block import TrustChainBlock` and `...attestation.trustchain.test_block import TestBlock`, PyCharm placed `...attestation.trustchain.test_block import TestBlock` first (and not together with the rest of the `attestation` imports) because it starts with `...` rather than `....`, that is 3 dots not 4 (or at least that's what I suspect).

Reply:
Please don't rely on PyCharm then in that case.

Reply:
Ok